docs/docs/single-node-network.md

```diff
@@ -43,9 +43,9 @@ For more information checkout the complete [documentation on keys](../cli-keys/)
 **Step 5 - Adding the validator to the network**
 
 To set the initial state for the network use the following command. This allocates `100000000000000000000` stake tokens
-to the validator which can be bonded.
+to the validator which can be bonded. We set a second token allocation of fet so that the account has funds to create transactions:
 
-   `fetchd add-genesis-account validator 100000000000000000000stake`
+   `fetchd add-genesis-account validator 100000000000000000000stake,9631fet`
```

Comment:
why? you can use `stake` tokens for any transactions. 

If using `stake` is confusing, we could replace it with `atestfet` if we want. All needed is to sed the genesis file to replace the denom everywhere (ie with `sed -i "s/\"stake\"/\"atestfet\"/" ~/.fetchd/config/genesis.json`, to be ran after the initial `fetchd init`)

Also, we may be bonding this whole `stake` amount at the moment, maybe we need to add some more (ie `fetchd add-genesis-account validator 1000000000000000000000000stake` in order to have some left on the account after the validator creation)

And last, we could add a 5.5 section or something, showing how to generate an extra "user" key, and funding this account: `fetchd keys add user`, `fetchd add-genesis-account user 100000000000000000000stake`)

Reply:
I'm just updating it to how I've figured out to use it. If I'm using is wrong, it's because I couldn't find this information in the docs. 

I couldn't find any info on what `stake` was in the docs, but that could be a search fault.

Reply:
`stake` is a token denomination, exactly as `afet`, `atestfet`, `BTC`, `uosmo`... are. This is the default testing denomination used by the whole cosmos ecosystem. 

Yes documentation is quite scarce, and given the pace of the ecosystem evolution, it'll often be outdated no matter the source. But luckily the code still doesn't lie :)

Reply:
Alrighty, good to know, I was using already developed scripts that deployed contracts but didn't use `stake`, they used `atestfet` 

I'll update this.